app/components/app-header.hbs

```diff
@@ -1,3 +1,15 @@
 <header class="app-header">
+  <button
+    class="app-header__mobile-nav-toggle"
+    aria-label="Show Site Navigation"
+    {{on "click" @toggleNavPrimary}}>
+    <svg aria-hidden="true" preserveAspectRatio="xMidYMid meet" height="1em" width="1em" fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" stroke="currentColor">
```

Comment:
If you want, you could add `role="image"` to this.

Reply:
Agreed!